data-es2016plus.js

```diff
@@ -5759,6 +5759,228 @@ exports.tests = [
       rhino1_7_13: false
     }
   },
+  {
+    name: '.at() method on the built-in indexables',
+    category: '2022 features',
+    significance: 'tiny',
+    spec: 'https://github.com/tc39/proposal-relative-indexing-method/',
+    subtests: [
+      {
+        name: 'Array.prototype.at()',
+        exec: function() {/*
+        var arr = [1, 2, 3];
+        return arr.at(0) === 1
+          && arr.at(-3) === 1
+          && arr.at(1) === 2
+          && arr.at(-2) === 2
+          && arr.at(2) === 3
+          && arr.at(-1) === 3
+          && arr.at(3) === undefined
+          && arr.at(-4) === undefined;
+      */},
+        res: {
+          ie11: false,
+          firefox68: false,
+          firefox85: firefox.nightly,
+          firefox90: true,
+          chrome85: false,
+          chrome92: true,
+          safari14_1: {
+            val: 'flagged',
+            note_id: 'safari-at-method',
+            note_html: 'The feature has to be enabled via <code>jscOptions=--useAtMethod=true</code> flag.'
+          },
+          safaritp: {
+            val: 'flagged',
+            note_id: 'safari-at-method'
+          },
+          babel7corejs3: babel.corejs,
+          typescript4corejs3: typescript.corejs,
+          graalvm21: graalvm.es2022flag,
+          rhino1_7_13: false
+        }
+      },
+      {
+        name: 'String.prototype.at()',
+        exec: function() {/*
+        var str = 'abc';
+        return str.at(0) === 'a'
+          && str.at(-3) === 'a'
+          && str.at(1) === 'b'
+          && str.at(-2) === 'b'
+          && str.at(2) === 'c'
+          && str.at(-1) === 'c'
+          && str.at(3) === undefined
+          && str.at(-4) === undefined;
+      */},
+        res: {
+          ie11: false,
+          firefox68: false,
+          firefox85: firefox.nightly,
+          firefox90: true,
+          chrome85: false,
+          chrome92: true,
+          safari14_1: {
+            val: 'flagged',
+            note_id: 'safari-at-method'
+          },
+          safaritp: {
+            val: 'flagged',
+            note_id: 'safari-at-method'
+          },
+          graalvm21: graalvm.es2022flag,
+          rhino1_7_13: false
```

Comment:
Please, add `core-js` to those results.

Reply:
is it enabled by default for both Array and %TypedArray%?

Reply:
And for `String` that's missed in this PR.